masonry/src/util.rs

```diff
@@ -15,15 +15,14 @@ use std::hash::Hash;
 ///
 /// It's useful when a backtrace would aid debugging but a crash can be avoided in release.
 macro_rules! debug_panic {
-    () => { ... };
-    ($msg:expr) => {
+    // () => { ... }; // TODO: What was this rule for?
```

Comment:
```suggestion
```

No idea where that's from.

Since it's not used anywhere and I'm pretty sure it wouldn't compile anyway, we can remove it.